src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -733,14 +733,22 @@ function octetStringEscapedForCLiteral(value)
 {
   // Escape control characters, things outside the ASCII range, and single
   // quotes (because that's our string terminator).
-  return value.replace(/\p{Control}|\P{ASCII}|"/gu, ch => {
-    let code = ch.charCodeAt(0);
-    code     = code.toString(16);
+  var notASCII = [...value ].some(char => 32 > char.charCodeAt(0) || char.charCodeAt(0) > 126 || char == "\"");
+  if (!notASCII) {
+    return value
+  }
+
+  // Since there is a character that is not ASCII, convert all to string literal.
```

Comment:
This will make various just-happens-to-be-non-ASCII-but-is-mostly-readable strings in our tests much harder to read...

Some options for what we could do to minimize that problem:
1. Use octal escapes instead of hex.  That does make it a little harder to tell what the escaped value is.
2. Use `" "\xhh" "` to escape a single digit tat needs escaping (i.e. instead of `return "\\x" + code;` in the original code just do `return `" "\\x${code}" "`;` in the original code we had here.

Thoughts?

Reply:
I think this would be much cleaner when viewing the generated code from YAML. Let me give it a shot with various scenarios to make sure it works with large payloads with mixing readable characters and non-readable characters.

Reply:
Second option does not work when there is a mix of escape and ASCII characters. The ASCII characters don't have the extra quotes so they get left out. It appears the octal format seems to work with the mixture of ASCII and non readable characters.

Reply:
Updated to octal.